packages/api/src/command/medical/cq-directory/parse-cq-directory-entry.ts

```diff
@@ -23,34 +23,27 @@ export function parseCQDirectoryEntries(orgsInput: Organization[]): CQDirectoryE
       return [];
     }
     const orgOid = org.identifier?.value?.value;
-    if (!orgOid) return [];
+    const normalizedOid = getOid(org, orgOid);
```

Comment:
nit: we're passing the org to the `getOid` function, so might as well have the retrieval of the OID there.